airbyte-integrations/connectors/source-salesloft/setup.py

```diff
@@ -5,7 +5,7 @@
 
 from setuptools import find_packages, setup
 
-MAIN_REQUIREMENTS = ["airbyte-cdk", "pendulum"]
+MAIN_REQUIREMENTS = ["airbyte-cdk==0.64.0", "pendulum"]
```

Comment:
Question: why this specific version? Is it the latest one?

Reply:
This is the `current` one in our branch. If we merge upstream, this will change.